tests/codegen/parseViewFunctions.test.ts

```diff
@@ -0,0 +1,77 @@
+import { expect, test } from "bun:test";
```

Comment:
Why does this test code have to depend on bun?

Reply:
The test framework's vendor is bun, like jest. This is not the bun runtime.

Reply:
Praise The Maker.